src/services/ElementStatusEvents.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace lsst\nextbuilds\services;
+
+use yii\base\Component;
+use craft\base\Element;
+use craft\events\ElementEvent;
+use lsst\nextbuilds\behaviors\ElementStatusBehavior;
+
+class ElementStatusEvents extends Component
```

Comment:
The service class that either registers the behavior and contains the methods that are called upon the event firing